docs/Quickstart_wuf.md

```diff
@@ -0,0 +1,145 @@
+---
+layout: page
+---
+
+# TO Do Service Quick Start <!-- omit in toc -->
+
+The **To Do Service API** is a cloud-based service that lets subscribers post to and review their task lists, and recieve reminders of tasks they'll need to complete. To start using this service, all you need is a user and a task!  
+This quick start should take **15 minutes** to complete.
+
+- [What you will learn](#what-you-will-learn)
+- [Before you begin](#before-you-begin)
+- [Users and Tasks](#users-and-tasks)
+- [Recap](#recap)
+- [Learn more](#learn-more)
+
+## What you will learn
+
+- [How to add a user.](#enroll-a-new-user)
+- [How to add tasks.](#add-tasks)
+- [Get a list of tasks for a specific user.](#get-the-tasks-for-a-user)
+
+## Before you begin
+
+To sucessfully complete this quick start you will need to set up a test environment. Be sure to complete the [Before you start a tutorial](before-you-start-a-tutorial.md) before you start this quick start.
+
+## Users and Tasks
+
+The **To Do Service API** uses JSON files to create its database. In this tutorial, we will use cURL to `POST` and `GET` our data, but you can use any other client, such as Postman to send and receive data.
+
+### Enroll a new user
+
+Enrolling a new user in the service requires that you add `POST` the details of a new [`user`](../api/user.md) resource to the service. The only information you need to add a user is
+
+To enroll a new user:
+
+1. Make sure your local service is running, or start it by using this command, if it's not.
```

Comment:
Do they know how to make sure their local service is running?

It might be easier to read the directions if this was broken into two. (Not everyone reads beyond the first few words in the line.)

```suggestion
1. If the To-Do service is not running, start it by using this command in your command-line tool.
```

This suggestion assumes they know how to see if the service is running.